rackcity/views/user_views.py

```diff
@@ -37,10 +37,9 @@
 from rest_framework.permissions import IsAuthenticated
 
 
-class RegisterNameView(PermissionRequiredMixin, RegisterView):
+class RegisterNameView(RegisterView):
+    permission_classes = IsAuthenticated, RegisterUserPermission
```

Comment:
overwriting the permission_classes instead allows us to override some of the behavior happening in the `django-rest-auth` library